example/satosa/openid4vci_frontend.yaml

```diff
@@ -0,0 +1,677 @@
+      session:
+        timeout: 6
+      verify: false
+
+  jwt:
+    default_sig_alg: &default_sig_alg ES256  # or RS256. Please note that this signature alg MUST be compliant with the private keys used for the signature. X.509 certificates MUST be therefore ECDSA using ES, and RSA using RS
+    default_enc_alg: RSA-OAEP
+    default_enc_enc: A256CBC-HS512
+    default_exp: 6         # minutes
+    enc_alg_supported:
+      - RSA-OAEP
+      - RSA-OAEP-256
+      - ECDH-ES
+      - ECDH-ES+A128KW
+      - ECDH-ES+A192KW
+      - ECDH-ES+A256KW
+    enc_enc_supported:
+      - A128CBC-HS256
+      - A192CBC-HS384
+      - A256CBC-HS512
+      - A128GCM
+      - A192GCM
+      - A256GCM
+    sig_alg_supported:
+      - RS256
+      - RS384
+      - RS512
+      - ES256
+      - ES384
+      - ES512
+    access_token_exp: 90 # minutes
+    refresh_token_exp: 120 # minutes
+    par_exp: 90 # minutes
+
+  # private jwk
+  metadata_jwks: &metadata_jwks
+    - kty: EC # Please note: this is the first key [0] and it is used for signing the presentation requests
+      d: i0HQiqDPXf-MqC776ztbgOCI9-eARhcUczqJ-7_httc
+      use: sig
+      crv: P-256
+      kid: f10aca0992694b3581f6f699bfc8a2c6cc687725
+      x: jE2RpcQbFQxKpMqehahgZv6smmXD0i/LTP2QRzMADk4
+      y: qkMx5iqt5PhPu5tfctS6HsP+FmLgrxfrzUV2GwMQuh8
+      alg: ES256
+
+  #This is the configuration for the relaying party metadata
+  metadata:
+    oauth_authorization_server:
+      issuer: https://eaa-provider.example.org # this field if not set will be autopopulated using internal variables base_url and name using the following format: "<base_url>/<name>"
+      pushed_authorization_request_endpoint: /as/par
+      authorization_endpoint: /authorize
+      token_endpoint: /token
+      client_registration_types_supported:
+        - automatic
+      code_challenge_methods_supported:
+        - S256
+      acr_values_supported:
+        - https://trust-registry.eid-wallet.example.it/loa/substantial
+        - https://trust-registry.eid-wallet.example.it/loa/high
+      scopes_supported:
+        - EuropeanDisabilityCard
+        - mDL
+      response_modes_supported:
+        - form_post.jwt
+        - query
+      response_types_supported:
+        - code
+      authorization_signing_alg_values_supported:
+        - ES256
+        - ES384
+        - ES512
+      grant_types_supported:
+        - authorization_code
+      token_endpoint_auth_methods_supported:
+        - attest_jwt_client_auth
+      token_endpoint_auth_signing_alg_values_supported:
+        - ES256
+        - ES384
+        - ES512
+      request_object_signing_alg_values_supported:
+        - ES256
+        - ES384
+        - ES512
+      jwks: *metadata_jwks
+    openid_credential_issuer: &openid_credential_issuer_metadata
+      credential_issuer: https://eaa-provider.example.org # this field if not set will be autopopulated using internal variables base_url and name using the following format: "<base_url>/<name>"
```

Comment:
this must match the base url and the frontend name ... example.org is not usable